### PR TITLE
Element.style() now works with loadFont() #1003

### PR DESCRIPTION
**Pull request by DarkPrince304**
@futuremarc Since using a custom font-family on an HTML element requires it to be defined in @font-face this PR will create a @font-face inside a `<style>` tag (inside `<head>`).

But this works correctly only on a running server (like python SimpleHTTPServer) , otherwise it runs into cross-origin issues.

Here's how it can be used:
```javascript
myFont = loadFont('assets/Avenir.otf');
```
How to apply it on another element?
```javascript
myDiv.style("font-family", "Avenir");
```

Another Example
```javascript
myFont = loadFont('assets/Italic.otf');
```
How to apply it on another element?
```javascript
myDiv.style("font-family", "Italic");
```

Basically, I have used regular expression to extract out "font-name" from `<font-name>.otf` (.ttf, .woff,etc) to be used as `font-family` .

### CODE REVIEW

**Comment by futuremarc** [comment]
Great! I was going to crack this over weekend, thanks!

**Comment by DarkPrince304** [comment]
@futuremarc Happy to help! 
Could you also tell me if it works as expected on your side as well?

**Comment by lmccart** [comment]
this is great @DarkPrince304, thank you! @futuremarc let me know when you've had a chance to review and if all looks good I'll merge it in.

**Comment by dhowe** [comment]
Nice...

**Comment by lmccart** [comment]
this is great!
